Description: Fix Django 1.7 compat
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-09-09

--- python-coffin-0.3.8.orig/tests/__init__.py
+++ python-coffin-0.3.8/tests/__init__.py
@@ -5,6 +5,11 @@ import sys
 # module code rather than setup_package(), because we want it to run
 # before any module-wide imports in any of the test modules.
 sys.path.insert(0, path.join(path.dirname(__file__), 'res', 'apps'))
-from django.core.management import setup_environ
-import settings
-setup_environ(settings)
+
+import os
+import django
+from templatelibs_app import models
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tests.res.apps.settings")
+if hasattr(django, 'setup'):
+    django.setup()
